docs/src/upgrade/release-3.4.x.asciidoc

```diff
@@ -28,3 +28,8 @@ NEED AN IMAGE
 Please see the link:https://github.com/apache/tinkerpop/blob/3.4.0/CHANGELOG.asciidoc#release-3-4-0[changelog] for a complete list of all the modifications that are part of this release.
 
 === Upgrading for Users
+
+==== Change in order of select() scopes
+
+The order of select scopes has been changed to: maps, side-effects, paths
+Previously the order was: side-effects, maps, paths - which made it almost impossible to select a specific map entry if a side-effect with the same name existed.
```

Comment:
two things: 

1. Could you go the extra step and add an example of the behavior before/after the change. I think people would understand it better if they could see the example (remember the source in upgrade is static so you should just copy/paste the example from your console)

2. note that we typically add a link to the relevant JIRA ticket: https://github.com/apache/tinkerpop/blame/master/docs/src/upgrade/release-3.3.x.asciidoc#L53